chat/fe/src/components/Input.ts

```diff
@@ -1,3 +1,7 @@
 export function Input() {
   return document.createElement("input");
+}
+
+export function InputArea() {
```

Comment:
This function should be called TextArea and be moved to it's own TextArea.ts file